docs/GraFx-Studio/guides/template-variables/define/index.md

```diff
@@ -65,34 +69,34 @@ A placeholder for a reference to an image in the (GraFx) Media pool, or other so
 
 ## Default state
 
-The default value your variable with have.
+The default value your template variable with have.
 
 ## Basic properties:
 
 - Name: Cannot be empty, must be unique
-- Type: See [types](#variable-types) above
+- Type: See [types](#types) above
 - [Visible](#visibility-conditions): Enabled by default
-- [Required](#required-variables): Disabled by default
+- [Required](#required-template-variables): Disabled by default
 - Read-only: Disabled by default
 
-### Required variables
+### Required template variables
 
 When checked, the Studio Editor will ask the user to provide a value.
 
 !!! Info
-	When a variable is required and it doesn't have a default value it doesn't show an error upon opening the document.
+	When a template variable is required and it doesn't have a default value it doesn't show an error upon opening the document.
```

Comment:
You cannot set a variable as required yet in Studio. You can only toggle the visibility as of now.